### ISSUE TITLE
[BUG] SCNvimHelp not working in the lua-config branch

### ISSUE THREAD

**Issue by kflak** [davidgranstrom/scnvim#185]
**Describe the bug**

Hit K on a Class produces this error:
```
Open ended string started on line 1 of interpreted text
ERROR: syntax error, unexpected $end, expecting ')'
  in interpreted text
  line 1 char 36:
  HelpBrowser.openHelpFor(\"SinOsc\") 
                                      
-----------------------------------
ERROR: Command line parse failed
```

**Expected behavior**

Help should work as per usual...

**Steps to reproduce**

Shift-K on a word. Also tried running `:SCNvimHelp SinOsc` and got the same result.

**Additional context**

**Information**

- Operating system

Arch Linux 5.15.36.41.realtime1-2-rt-lts

- SuperCollider version

3.12.2

- `nvim --version`

NVIM v0.7.0
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by builduser

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info

- Package manager

packer.nvim

**Comment by kflak** [comment]
OK, found the bug:

Line 187 in help.lua needs to be changed from:
```
    local cmd = string.format([[HelpBrowser.openHelpFor(\"%s\")]], subject)
```
to
```
    local cmd = string.format([[HelpBrowser.openHelpFor("%s")]], subject)
```

**Comment by kflak** [comment]
However, it opens up the qt window, not a split window as before. Is this to be expected? Don't mind the qt browser, tbh, but it's nice to have  the option...

**Comment by davidgranstrom** [comment]
@kflak Thanks for the report, should be fixed now with 367b4e3d9fe052e03ef89263b58832a99bd2df5b

To use the plain text help system you will need to specify the command used for the html -> plain text conversion (e.g. `pandoc`)

Add the following to `scnvim.setup` (adjust `cmd` according to your enviroment)
```
  documentation = {
    cmd = '/opt/homebrew/bin/pandoc',
  },
```

**Comment by kflak** [comment]
Great, works now on my system. Also plaintext works as expected with the modification to the config. I must say I am very excited about this rewrite!
